dom/nodes/aria-element-reflection.tentative.html

```diff
@@ -342,10 +344,12 @@
   test(function(t) {
     const billingElement = document.getElementById("billingElement")
     assert_array_equals(input1.ariaLabelledByElements, [billingElement, nameElement], "parsed content attribute sets element references.");
+    assert_array_equals(input1.ariaLabelledByElements, input1.ariaLabelledByElements, "check idl attribute caching after parsing");
```

Comment:
These need to be `assert_equals`, not `assert_array_equals`.

Reply:
I've done the change, but now both Chromium and WebKit implementations fail this check.

Reply:
Somehow expected it fails in Chromium given this coment:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/element.cc;l=995
```cpp
  // TODO(chrishall): this will fail to preserve `e1.ariaFoo === e1.ariaFoo`,
  // need additional cache to preserve this invariant, add tests covering this
  // case.
```

Though on a quick sight I'm not really sure how that cache is going to be implemented, as this is implemented using weak pointers to avoid memory issues, and the final array returned doesn't use weak pointers. So the one that would need to be cached won't use weak pointers I guess, which doesn't seem right.

Reply:
Ah OK great, I'm glad we caught this pretty critical issue before proceeding further with the spec.

This might be a reason to transition to `ObservableArray<T>` instead, since that preserves the invariant by construction.